Utilities/File.h

```diff
@@ -733,9 +733,18 @@ namespace fs
 		// Always use write flag, remove read flag
 		if (fs::file f{path, mode + fs::write - fs::read})
 		{
-			// Write args sequentially
-			(f.write(args), ...);
-			return true;
+			if constexpr ((std::is_pointer_v<std::remove_reference_t<Args>> || ...))
```

Comment:
In that case, you don't even need to remove reference or const I think.

Reply:
What about specisifying reference type explicitly in template like fs::write_file<const u32&> ?

Reply:
Why would someone need to do this? If it's to break to the else path, then let it break. Such arg packs are not to be explicitly declared in most cases.

Reply:
Fixed.